docs/installation.md

```diff
@@ -17,6 +17,14 @@ On other platforms, install `jrnl` using [Python](https://www.python.org/) 3.6+
 pipx install jrnl
 ```
 
+!!! note
+     `pipx` should be installed through `pip`. Missing dependencies and other issues
+     may occur when installing `pipx` through `apt` or another package manager.
+     
+!!! note
```

Comment:
Could this be a tip instead of a note?

Reply:
Just for clarification, are you referring to the first note being changed to a tip, the second note being changed to a tip, or both?